admin/lib/generators/solidus_admin/install/install_generator.rb

```diff
@@ -4,7 +4,11 @@ module SolidusAdmin
   module Generators
     class InstallGenerator < Rails::Generators::Base
       def install_solidus_core_support
```

Comment:
As we commented offline with @elia, we need to make sure routes are mounted before the ones for `Spree::Core::Engine`.